dts/bindings/ospi/st,stm32-ospi.yaml

```diff
@@ -33,3 +36,33 @@ properties:
 
     pinctrl-names:
       required: true
+
+    dmas:
+      description: |
+        Optional DMA channel specifier, required for DMA transactions.
+        For example dmas for TX/RX on OSPI
+           dmas = <&dma1 5 41 0x10000>;
+
+        With, in each cell of the dmas specificier:
+        - &dma1: dma controller phandle
+        - 5: channel number (0 to Max-Channel minus 1). From 10 to 15 on stm32u5x.
+        - 41: slot number (request which could be given by the DMAMUX)
+        - 0x10000: channel configuration (only for srce/dest data size, priority)
+
+        Notes:
+          - On series supporting DMAMUX, transnational DMA phandle should
```

Comment:
transnational ?

Reply:
fixed

Reply:
Did you forget to squash or push the fix? I still see it here